src/filetypes.c

```diff
@@ -951,7 +951,7 @@ static void load_settings(guint ft_id, GKeyFile *config, GKeyFile *configh)
 	{
 		ft->lang = tm_source_file_get_named_lang(result);
 		if (ft->lang < 0)
-			geany_debug("Cannot find tag parser '%s' for custom filetype '%s'.", result, ft->name);
+			geany_debug("Cannot find tags parser '%s' for custom filetype '%s'.", result, ft->name);
```

Comment:
Personally I prefer the "tags parser" better because this is closer to ctags and the parser is in fact a ctags parser (well, with some modifications at least for now). It could even be called "ctags parser" but we already use "tags file" so "tags parser" is more consistent with the used terminology.